src/pages/users/management/UserManagementTable.module.scss

```
@@ -0,0 +1,134 @@
+    display: flex;
+    flex-shrink: 0;
+    align-items: center;
+}
+
+.popoverTrigger {
+    display: inline-flex;
+}
+
+.sectionCount {
+    flex-shrink: 0;
+    font-family: 'Inter', 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)